default.nix

```diff
@@ -1,14 +1,12 @@
 {pkgs ? import <nixpkgs> {}}:
-pkgs.rustPlatform.buildRustPackage rec {
+pkgs.rustPlatform.buildRustPackage {
   pname = "nyaa";
-  version = "0.9.0";
-
-  # src = fetchFromGitHub {
-  #   owner = "Beastwick18";
-  #   repo = "nyaa";
-  #   rev = "v${version}";
-  #   hash = "sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=";
-  # };
```

Comment:
This section is needed only in nixpkgs when you publish it there, so in https://github.com/NixOS/nixpkgs

(I wasn't clear enough in previous issue discussion).